### PR TITLE
Ep/add resnext101 32x48d wsl

### PR DESCRIPTION
**Pull request by pellegreene**

### CODE REVIEW

File: brainscore_vision/models/resnext101_32x48d_wsl/model.py [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+from brainscore_vision.model_helpers.activations.pytorch import load_preprocess_images
+import torch.hub
+import ssl
+from brainscore_vision.model_helpers.check_submission import check_models
+
+
+ssl._create_default_https_context = ssl._create_unverified_context
+
+
+def get_model_list():
```
Comment (mike-ferguson) [inline brainscore_vision/models/resnext101_32x48d_wsl/model.py:12]: Can remove this

**Review by mike-ferguson — CHANGES_REQUESTED** [review]
lgtm once get model_list is removed

File: brainscore_vision/models/resnext101_32x48d_wsl/model.py (hunk 1 above)
Comment (pellegreene) [reply brainscore_vision/models/resnext101_32x48d_wsl/model.py]: Changes pushed!

**Comment by mike-ferguson** [comment]
Force merged per @pellegreene due to jenkins updates

**Pull request merged**